docs/Processes.md

```diff
@@ -1435,3 +1435,200 @@ and are under the `.ctp` namespace.
  | schema               |    retrieve schema from tickerplant    |                                                                   `1b`|
  | clearlogonsubscription  | clear log on subscription, only called if createlogfile is also enabled  |                              `0b`|
 
+TorQ Data Quality System Architecture
+-------
+
+The Data Quality System consists of two processes that are listed below: 
+Data Quality Checker (DQC) and Data Quality Engine(DQE). The Purpose of 
+the Data Quality System is to ensure quality of data in the data capturing
+system by running checks on other TorQ processes. The system behaves based
+on Data Quality Config files. The metrics from the config files and the
+results of the checks performed on databases in the data capturing system
+are saved to Data Quality Databases. The results are then used for monitoring
+tools to alert the users.
+
+
+Data Quality Checker (DQC)
+-------
+
+The Data Quality process is a process to run checks on other TorQ
+processes to ensure quality of data in the system. The Checker runs
+periodic checks on specific processes. The specific parameters of 
+the checks that are being run can be configured in `config/dqcconfig.csv`.
+Configuration from `dqcconfig.csv` are loaded to `configtable` in `dqe`
+namespace, and the checks are then run based on the parameters. The results
+of the checks are then saved to the results table in `dqe` namespace. The 
+results table that contains all check results would periodically be saved to
+Data Quality Checker Database (DQCDB) intraday. The configuration of the
+checks will also be periodically saved to the Data Quality Checker Database 
+intraday.
+
+Example of `configtable` is shown below:
```

Comment:
I think this may flow better if you do config file then results after so it flows